docs/conf.py

```diff
@@ -125,9 +125,6 @@ def build_mono_repo(temp_repo_folder):
 #
 
 html_theme = "furo"
-html_theme_options = {
-    "use_download_button": False,
-}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question (non-blocking):** ‏Did you mean to change this too?

Reply:
Yeah did some refactoring, that was an option in the old book theme but not in this theme so it was throwing warnings